compiler/rustc_lint/src/builtin.rs

```diff
@@ -2273,7 +2273,7 @@ impl EarlyLintPass for IncompleteInternalFeatures {
                     cx.emit_spanned_lint(
                         INTERNAL_FEATURES,
                         span,
-                        BuiltinInternalFeatures { name, note },
+                        BuiltinInternalFeatures { name, note: None }, // Passing 'None' as 'BuiltinInternalFeatures' struct already has '#[note]' attr
```

Comment:
```suggestion
                        BuiltinInternalFeatures { name },
```
If `note` is always `None`, then the corresponding field should be removed from the structure.

Reply:
Thanks. Done.